server.js

```diff
@@ -63,11 +63,41 @@ async function handleEvent(event) {
 
     return client.replyMessage(event.replyToken, {
       type: "text",
-      text: `${event.message.text} さんですね。次は学類を教えてください。`,
+      text: `${answer["name"]} さんですね。次は学類を教えてください。`,
     });
   }
 
   // こんな感じで順番に聞いてく
+
+  // 名前が記録され学類が入力されていないなら、入力されたメッセージは学類
+  if (answer["name"] != null && answer["faculties"] == null) {
+    // 学類を記録
+    answer["faculties"] = event.message.text;
+
+    return client.replyMessage(event.replyToken, {
+      type: "template",
+      template: {
+        type: "bottons",
+        text: `${answer["name"]}さん、${answer["faculties"]} 所属で登録してよろしいですか？`,
```

Comment:
`template` オブジェクトのプロパティは、`type`, `text`, `actions` 以外に `altText` が必須です！
```suggestion
      template: {
        type: "bottons",
        text: `${answer["name"]}さん、${answer["faculties"]} 所属で登録してよろしいですか？`,
        altText: `${answer["name"]}さん、${answer["faculties"]} 所属で登録してよろしいですか？ 「はい」か「いいえ」で教えてください。`,
```